app/com.zoop.zoopPC/run-java.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+exec java -jar /app/bin/ZoopPC-Server-jvm.jar
+
+# Definir permissões para o arquivo .jar
+echo "Definindo permissões para ZoopPC-Server.jar..."
+chmod 644 ZoopPC-Server-jvm.jar
+# Definir permissões para o script de execução
+echo "Definindo permissões para run-java.sh..."
+chmod +x run-java.sh
+
+echo "Permissões definidas com sucesso."
```

Comment:
```suggestion
```

Reply:
also there is no java in the runtime and you haven't taken the step to add it.